crec_stager/crec_stager.py

```diff
@@ -0,0 +1,258 @@
+            logging.info('No zip found for date {0}'.format(self.dt))
+            return None
+        logging.info(
+            'Extracting html files from zip to {0}'.format(self.zip_download_dir)
+        )
+        html_file_paths = self.extract_html_files(zip_path)
+        logging.info('Uploading {0} html files...'.format(len(html_file_paths)))
+        for file_path in html_file_paths:
+            try:
+                s3_key = self.upload_to_s3(file_path)
+            except ClientError as e:
+                logging.exception(
+                    'Error uploading .htm file {0}, exiting'.format(file_path, e)
+                )
+                return False
+        logging.info('Uploads finished.')
+        return True
+
+
+def lambda_handler(event, context):
+    """Entry point for AWS Lambda execution.
+
+    In addition to the arguments specified below, this function also gets some
+    settings from the following environment variables (set through the AWS
+    console):
+        LOGLEVEL
+            loglevel for logging to cloudwatch
+        ZIP_DOWNLOAD_DIR
+            what directory to download and unpack CREC zips. Must be under
+            ``/tmp`` as everything else is write protected in lambda.
+        S3_TARGET_BUCKET
+            what s3 bucket to upload unpacked html files to.
+
+    Args:
+        event (:obj:`dict`): A dictionary containg data from event trigger.
+        context (:obj:`dict`): Context settings for this lambda job.
+    """
+    logger = logging.getLogger()
+    logger.setLevel(os.environ.get('LOGLEVEL', 'INFO'))
+    formatter = logging.Formatter(DEFAULT_LOG_FORMAT)
+    zip_download_dir = os.environ.get('ZIP_DOWNLOAD_DIR', '/tmp')
+    s3_bucket = os.environ.get('S3_TARGET_BUCKET')
+    if not s3_bucket:
+        raise Exception('No s3 bucket defined in $S3_TARGET_BUCKET.')
+    s3_key_prefix = os.environ.get('S3_KEY_PREFIX', 'capitolwords/')
+    crec_stager = CRECStager(
+        datetime.utcnow() - timedelta(days=1),
+        zip_download_dir,
+        s3_bucket,
+        s3_key_prefix
+    )
+    crec_stager.stage_html_files()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--date',
+        help='Date to retrieve records for, format is YYYY-MM-DD.',
+        type=lambda d: datetime.strptime(d, CMD_LINE_DATE_FORMAT),
+    )
+    parser.add_argument(
+        '--s3_bucket',
+        help='Bucket to upload html files to.',
+        default='use-this-bucket-to-test-your-bullshit',
+    )
+    parser.add_argument(
+        '--s3_key_prefix',
+        help='Key prefix for the html files staged in S3.',
+        default='capitolwords/',
+    )
+    parser.add_argument(
+        '--zip_download_dir',
+        help='Directory to write the zip and extracted files to.',
+        default='/tmp'
+    )
+    parser.add_argument(
+        '--loglevel',
+        help='Log level, one of INFO, ERROR, WARN, DEBUG or CRITICAL.',
```

Comment:
`choices=[logging.INFO, logging.ERROR, ...]` and then `default=logging.INFO`. then you can get rid of the LOGLEVELS dict. :)